handbook/product/design/design-process.md

```diff
@@ -100,12 +101,19 @@ While asynchronous communication is a core attribute of remote work, key moments
         - Interactive mode / plain text mode
   - Process
       - A GitHub issue should be created to track the work
-      - Wireframes are produced in Figma
-      - Wireframes will be announced in Slack and linked in the GitHub issue well before they are complete for review
-      - Schedule and conduct a research presentation meeting
+      - Designs are produced in Figma
+      - Designs will be announced in Slack and linked in the GitHub issue well before they are complete for review
+      - Ensure designs meet the visual design checklist:
+        - Ensure spacing is consistent and matches the 8pt grid system
+        - Ensure colors match existing styles
```

Comment:
How about mentioning the text styles here as well?

Ensure text styles match the existing ones, if possible. Introduce new text styles only if really necessary. Remember to add them to the design system with proper description and ensure that they meet the accessibility standards.

Reply:
Great addition, I generalized this comment to refer to all styles and added accessibility standards note.